jacob-george/starter-code/public/scripts/models/repo.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+var app = app || {};
+
+(function(module) {
+  const repos = {};
+
+  repos.all = [];
+
+  repos.requestRepos = function(callback) {
+    // DONE: How would you like to fetch your repos? Don't forget to call the callback.
+    //       Remember that the callback function we'll want to call relies on repos.all
+    //       being an array with a bunch of repo objects in it, so you'll need to
+    //       populate it with the response from Github before you call the callback.
+    $.get('/github/user/repos', function (repo){
+      repo.map(ele => {
```

Comment:
No need to map since the data we get back is already an array of objects(repos). A good way to do this is not with a callback function in the get request, but a .then promise that carriers has a success and failure function. Refer to the article.js $.get(/article) function it's pretty similar.